its/projects/RazorClassLib/S6797/S6797.NoRoute.razor

```diff
@@ -0,0 +1,90 @@
+﻿<p>
```

Comment:
Same as for the other ITs, I suggest keeping those minimal to make sure one issue is raised and let the UTs take care of the whole list of test cases

Reply:
I think this was not addressed even tho I see the thumbs up 👍